statements/downloaded_file.md

```diff
@@ -0,0 +1,35 @@
+# Downloaded a file
+
+## Description
+
+This event is emitted when a user has downloaded a file such as a pdf, doc, txt, ppt, xls, csv, etc.
+
+## Example
+
+```json
+{
+   "actor": {
+      "objectType": "Agent",
+      "account": {
+         "name": "john",
+         "homePage": "http://gaiax.org"
+      }
+   },
+   "verb": {
+      "id": "https://w3id.org/xapi/netc/verbs/downloaded"
+   },
+   "object": {
+      "objectType": "Activity",
```

Comment:
```suggestion
```
This field is not mandatory. We try to minimize the statement content. The definition type is enough to give information that this is an activity and not an agent